cfpb/cfgov-refresh
# Archive events

Created a script to be run on the cmdline by the jenkins cron job hourly to archive past events.

- @kurtw 
- @rosskarchner 
- @richaagarwal 

### Test
- import Event pages `./cfgov/manage.py import-data events eventpage --parent events -u <wagtail_admin_username> -p <wagtail_admin_pw>`
 - WP events should be living under `/events/`
- run archive events script `./cfgov/manage.py runscript archive_events`
 - WP events now live under `/events/archive`
